include/openmc/lattice.h

```diff
@@ -265,8 +269,15 @@ class HexLattice : public Lattice
   void to_hdf5_inner(hid_t group_id) const;
 
 private:
+  //! Fill universes_ vector for OY orientation
+  void fill_lattice_oy(std::vector<std::string> univ_words);
```

Comment:
Passing a vector by value will result in it being copied. Instead, pass a constant reference:
```C++
void fill_lattice_y(const std::vector<std::string>& univ_words);
```